Jenkinsfile_nightly

```diff
@@ -27,13 +27,22 @@ def secrets = [
                 secret('microservicekey-ccd-ps', 'PRINT_S2S_KEY'),
                 secret('microservicekey-ccd-admin', 'ADMIN_S2S_KEY')
         ],
+        'prl-${env}'      : [
+                secret('legalprofessional-testuser-one', 'LEGALPROFESSIONAL_TESTUSER_ONE'),
+                secret('legalprofessional-respondent-testuser', 'LEGALPROFESSIONAL_RESPONDENT_TESTUSER'),
+                secret('legalprofessional-testpassword-one', 'LEGALPROFESSIONAL_TESTPASSWORD_ONE'),
+                secret('courtadmin-testuser-one', 'COURTADMIN_TESTUSER_ONE'),
+                secret('courtadmin-swansea-testuser', 'COURTADMIN_SWANSEA_TESTUSER'),
+                secret('judge-testuser-one', 'JUDGE_TESTUSER_ONE'),
+                secret('judge-testpassword', 'JUDGE_TESTPASSWORD')
+        ],
         'ccd-${env}'      : [
                 secret('postcode-info-address-lookup-token', 'ADDRESS_LOOKUP_TOKEN'),
                 secret('ccd-api-gateway-oauth2-client-secret', 'API_GATEWAY_IDAM_SECRET'),
                 secret('ccd-admin-web-oauth2-client-secret', 'ADMIN_WEB_IDAM_SECRET'),
                 secret('idam-data-store-client-secret','IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET'),
-                secret('idam-data-store-system-user-username','IDAM_DATA_STORE_SYSTEM_USER_USERNAME'),
```

Comment:
Sorry @gokul-sol , dont we need the user name anymore?

Reply:
@AlokDatta good catch, added it now. Thanks.